phpBB/phpbb/storage/adapter/local.php

```diff
@@ -131,11 +151,12 @@ public function create_dir($path)
 	{
 		try
 		{
-			$this->filesystem->mkdir($path);
+			$this->filesystem->mkdir($this->root_path . $path);
 		}
 		catch (filesystem_exception $e)
 		{
 			throw new exception('', $path, array(), $e); // CANNOT_CREATE_DIRECTORY
```

Comment:
This branch is only for development... in the push request to the master branch this is ok